.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Magento Cloud Tools code
+
+Use the GitHub fork & pull model contribution model to submit your code contributions to the Magento Cloud Tools codebase.
+In this contribution model, you maintain your own [fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks) of the Magento Cloud Tools repository, and create a [pull request](https://help.github.com/articles/about-pull-requests/) to submit your proposed changes to the base repository. For details on the fork & pull contribution model, see the [Beginners guide](https://github.com/magento/magento2/wiki/Getting-Started).
```

Comment:
Is it okay that link goes to core repo?

Reply:
We don't have a wiki here, having DevDocs page may be reasonable but it would duplicate same docs

Reply:
The detailed contributor instructions for GitHub are on the magento2 wiki.  The _Beginners Guide_.  The Contributor's Guide is a DevDocs link and is more conceptual with links to the Beginners Guide for instructions.